.travis.yml

```diff
@@ -1,28 +1,18 @@
 language: c
 env:
-  - BD=esp8266:esp8266:nodemcuv2:xtal=80,eesz=4M3M,ip=lm2f,exception=disabled
-  # - BD=esp8266:esp8266:d1_mini:xtal=80,eesz=4M3M,ip=lm2f,exception=disabled
+  - PLATFORMIO_BUILD_CACHE_DIR="../../.pio/buildcache"
+cache:
+  directories:
+    - "~/.platformio"
 before_install:
-  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_1.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :1 -ac -screen 0 1280x1024x16"
-  - sleep 3
-  - export DISPLAY=:1.0
-  - wget http://downloads.arduino.cc/arduino-1.8.8-linux64.tar.xz
-  - tar xf arduino-1.8.8-linux64.tar.xz
-  - sudo mv arduino-1.8.8 /usr/local/share/arduino
-  - sudo ln -s /usr/local/share/arduino/arduino /usr/local/bin/arduino
   - wget https://raw.githubusercontent.com/google/styleguide/gh-pages/cpplint/cpplint.py
 install:
-  - ln -s $PWD /usr/local/share/arduino/libraries/
-  - git clone https://github.com/tzapu/WiFiManager.git /usr/local/share/arduino/libraries/WiFiManager
-  - git clone https://github.com/knolleary/pubsubclient.git /usr/local/share/arduino/libraries/PubSubClient
-  - git clone https://github.com/bblanchon/ArduinoJson.git --branch 6.x /usr/local/share/arduino/libraries/ArduinoJson
-  - arduino --pref "boardsmanager.additional.urls=http://arduino.esp8266.com/stable/package_esp8266com_index.json" --save-prefs
-  - arduino --install-boards esp8266:esp8266
-  - arduino --board $BD --save-prefs
-  - arduino --pref "compiler.warning_level=all" --save-prefs
   - sudo apt-get install jq
   - sudo apt-get purge python-enum34
   - sudo apt-get install pylint3
+  - sudo pip install platformio
+  - pio update
+  - pio lib -g install ArduinoJson@^5.0.0 PubSubClient WifiManager
```

Comment:
Is this line required still?
Those examples that require those dependencies should be included via their specific `.platformio.ini` files I would have thought. If they are not, then I should fix that.

Not something we need to fix right this second or this PR. Just a question.

Reply:
Confirmed in #939 that it is no-longer needed.